### ISSUE TITLE
[Bug]: Pulling docker image failed: failed to register layer

### ISSUE THREAD

**Issue by lmallin** [epi2me-labs/wf-human-variation#35]
### What happened?

A bug happened!

I'm trying to run this workflow on the DNAnexus cloud platform but it fails trying to pull the docker image:
`failed to register layer: ApplyLayer exit status 1 stdout:  stderr: lchown /models/r941_prom_sup_g5014: invalid argument`

According to DNAnexus support, this is because the host machine, including the cloud worker on DNAnexus, follows the 16-bit UID range (0-65535). It can cause issues with running "docker pull ..." commands when one or more layers / files were created by the user account(s) with UID or GID outside of that range from the builder's local system.

Their suggestion is to rebuild the docker with all UIDs/GIDs less than 65535 as a workaround and use the "--chown=" parameter to change the ownership of files in your dockerfile and rebuild the image (as per https://blog.mornati.net/docker-images-and-files-chown).

### Operating System

ubuntu 20.04

### Workflow Execution

Command line

### Workflow Execution - CLI Execution Profile

None

### Workflow Version

v1.3.0

### Relevant log output

```shell
Error executing process > 'snp:getVersions'
Caused by:
  Process `snp:getVersions` terminated with an error exit status (1)
Command executed:
  run_clair3.sh --version | sed 's/ /,/' >> versions.txt
Command exit status:
  1
Command output:
  (empty)
Command wrapper:
  4c7a1aa9978f: Pulling fs layer
  097aea170ad7: Pulling fs layer
  e0725ede2f94: Pulling fs layer
  e90e51ba0fe0: Pulling fs layer
  2aeb24a5c0c5: Pulling fs layer
  78417453c13a: Pulling fs layer
  cb044527db5a: Pulling fs layer
  097aea170ad7: Waiting
  e0725ede2f94: Waiting
  e90e51ba0fe0: Waiting
  2aeb24a5c0c5: Waiting
  78417453c13a: Waiting
  27a9f84bae05: Waiting
  0d9e0007c6ce: Waiting
  4c7a1aa9978f: Waiting
  cb044527db5a: Waiting
b3bf890f2514: Verifying Checksum
  b3bf890f2514: Download complete
  841971ce0eda: Verifying Checksum
  841971ce0eda: Download complete
  27a9f84bae05: Verifying Checksum
  27a9f84bae05: Download complete
  3b65ec22a9e9: Verifying Checksum
  3b65ec22a9e9: Download complete
  0d9e0007c6ce: Verifying Checksum
  0d9e0007c6ce: Download complete
  4c7a1aa9978f: Verifying Checksum
  4c7a1aa9978f: Download complete
  097aea170ad7: Verifying Checksum
  097aea170ad7: Download complete
  e90e51ba0fe0: Verifying Checksum
  e90e51ba0fe0: Download complete
  2aeb24a5c0c5: Verifying Checksum
  2aeb24a5c0c5: Download complete
  78417453c13a: Verifying Checksum
  78417453c13a: Download complete
  3b65ec22a9e9: Pull complete
  841971ce0eda: Pull complete
  b3bf890f2514: Pull complete
  27a9f84bae05: Pull complete
  e0725ede2f94: Verifying Checksum
  e0725ede2f94: Download complete
  0d9e0007c6ce: Pull complete
  4c7a1aa9978f: Pull complete
  097aea170ad7: Pull complete
  cb044527db5a: Verifying Checksum
  cb044527db5a: Download complete
  failed to register layer: ApplyLayer exit status 1 stdout:  stderr: lchown /models/r941_prom_sup_g5014: invalid argument
  Pulling docker image failed after 3 attempts
```

**Comment by SamStudio8** [comment]
Hi @lmallin, we believe this problem was identified in v1.3.0 and fixed in our recent v1.4.0 update. Can you update your version of the workflow with `nextflow pull epi2me-labs/wf-human-variation` and try again?

**Comment by lmallin** [comment]
Hi Sam, 

Thanks for getting back to me so quickly.  I've just checked with v1.4.0 and it works fine now so that's great, thank you!

(I must have missed the v1.4.0 update by a couple of hours as I only cloned last week!)
